lib/toolchain.rb

```diff
@@ -205,6 +188,10 @@ def install
     rm_r "mnt/install/lib/bfd-plugins"
     rm ["mnt/install/lib/libcc1.so", "mnt/install/lib/libcc1.0.so"]
     chmod "-w", "mnt/install/lib"
+    mkdir "mnt/install/bin"
+
+    brew_prefix = Pathname.new(ENV["HOMEBREW_PREFIX"] || "/usr/local/homebrew")
+    ln_s brew_prefix/"bin/lld", "mnt/install/bin/ld.lld"
```

Comment:
This looks sketchy. I think the standard pattern looks more like this: https://stackoverflow.com/a/73408649

Reply:
We do a similar thing to get lld support there: https://github.com/MaterializeInc/materialize/blob/92acf2387bb0ff7331d7d452857d399801fb314a/ci/builder/Dockerfile#L213-L214

Reply:
I don't think the stackoverflow thread's approach works to get lld working. I did find another approach though, so this PR might not be necessary to get the lld speedup.